components/server/ee/src/prebuilds/github-app.ts

```diff
@@ -329,11 +334,11 @@ export class GithubApp {
         return this.env.hostUrl.with({ pathname: '/button/open-in-gitpod.svg' }).toString();
     }
 
-    protected async findInstallationOwner(installationId: number): Promise<{user: User, project?: Project} | undefined> {
+    protected async findInstallationOwner(installationId: number, cloneURL: string): Promise<{user: User, project?: Project} | undefined> {
```

Comment:
Should this rather be two methods? (findUser(installationId) and findPRoject(cloneUrl))
Maybe at least name the method `findInstallationOwnerAndProject`

Reply:
both make sense. will do!